### ISSUE TITLE
napisy24.pl Login failure

### ISSUE THREAD

**Issue by StarSmasher44** [morpheus65535/bazarr#1092]
**Describe the bug**
One of the providers, napisy24.pl keeps giving me a login error/authentication failed.
error message: Throttling napisy24 for 10 minutes, until 20/08/28 18:21, because of: AuthenticationError. Exception info: 'Login failed'

I am sure the password is correct on both website and bazarr, as I checked both a few times.

**To Reproduce**
Add Napisy24.pl with username and password, hit save, error shows in provider page.

**Expected behavior**
Expecting it to not fail the login, as the details are correct.

**Software (please complete the following information):**
Windows Server, but I assume software is irrelevant for this case.

**Additional context**
Perhaps there is something wrong with the way it authenticates?

**Comment by morpheus65535** [comment]
Dude, I do I take the time to write information like this just under the username field:

`The Provided Credentials Must Have API Access. Leave empty to use the defaults.`

Did you at least ask for this access to napisy24 admins?

**Comment by StarSmasher44** [comment]
> 
> 
> Dude, I do I take the time to write information like this just under the username field:
> 
> `The Provided Credentials Must Have API Access. Leave empty to use the defaults.`
> 
> Did you at least ask for this access to napisy24 admins?

My apologies, I did read the note but I was unsure of how to take care of it, as I do not actually have the capability to read polish on my own. I was under the impression that it would not work at all if I did not fill in my information. But if it actually does work without my own account then I can just do that.
(Although, looking at the providers tab it now went back to 'Good', but if no credentials is better here then I'll prefer that)
